Add example of POSTing data to retrieve sensors

cvwillegen opened the pull request:
I found it hard to get a website that required POST data to work, so to save others the work of looking up how to specify the data (in this case) and setting the correct headers, I made this real-life example.

## Proposed change
Adding an example of how to retrieve data by specifying how to create a configuration that would send a POST request to a web site in order to retrieve data.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

cvwillegen committed:
Add example of POSTing data to retrieve sensors

I found it hard to get a website that required POST data to work, so to save others the work of looking up how to specify the data (in this case) and setting the correct headers, I made this real-life example.

cvwillegen committed:
Merge branch 'current' into patch-2

cvwillegen committed:
Merge branch 'current' into patch-2

cvwillegen committed:
Merge branch 'current' into patch-2

fabaff commented:
The example show something that is not covered in the examples yet. Still I think that the "Community Guides" at https://community.home-assistant.io/ is a better place for it. Most users don't speak Dutch and without a valid address users are not able to play with the sensor and learn.

cvwillegen committed:
Merge branch 'home-assistant:current' into patch-2

cvwillegen committed:
Fix example to retrieve actual data

I've added a working postcode/house number combination, so that users can play around with live data.

cvwillegen commented:
I've updated the example to yield working data (by searching for a working postal code/house number combination). I still think this is a better place instead of in the community guides, since it covers basic usage, and the example is basic enough to help people to adapt to their own web site APIs using the ``POST`` method.

frenck commented on source/_integrations/scrape.markdown:
```diff
@@ -263,3 +263,46 @@ scrape:
+scrape:
+  - resource: https://crm.cleanprofs.nl/search/planning
+    method: POST
+    payload: zipcode=5624JW&street_number=17
```
Why are the resource duplicated? You can call a post to a single resource once, and add multiple sensors to that single resource.

This right now, posts 4 times for no reason.

../Frenck

cvwillegen committed:
Removed duplicate resources

cvwillegen committed:
Merge branch 'current' into patch-2

cvwillegen committed:
Merge branch 'current' into patch-2

cvwillegen commented:
As far as I can tell, this is good to go now...

gjohansson-ST commented on source/_integrations/scrape.markdown:
```diff
@@ -259,3 +259,28 @@ scrape:
+
+### Container cleaning by CleanProfs in The Netherlands
+
+The example gets the container type and container cleaning date for the next two cleanings
```
```suggestion
This example gets the container type and container cleaning date for the next two cleanings.
```

gjohansson-ST commented on source/_integrations/scrape.markdown:
````diff
@@ -259,3 +259,28 @@ scrape:
+The example gets the container type and container cleaning date for the next two cleanings
+
+```yaml
+# Configuration entry. Change postal code and house number to your own address.
````
```suggestion
# Example configuration.yaml entry. Change postal code and house number to your own address.
```

gjohansson-ST committed:
Update source/_integrations/scrape.markdown

gjohansson-ST committed:
Update source/_integrations/scrape.markdown
